iree/base/api.c

```diff
@@ -953,7 +976,9 @@ iree_status_to_string(iree_status_t status, char** out_buffer,
                                         /*buffer=*/NULL, &buffer_length))) {
     return false;
   }
-  char* buffer = (char*)malloc(buffer_length + 1);
+  // add space for NULL and pass this information on to iree_status_format
```

Comment:
proper punctuation on comments here and elsewhere (`// Add a space for .... to iree_status_format.`) https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar